modules/hostap/src/supp_api.c

```diff
@@ -309,6 +311,14 @@ static int wpas_add_and_config_network(struct wpa_supplicant *wpa_s,
 		_wpa_cli_cmd_v("set_network %d mode 2", resp.network_id);
 	}
 
+	strncpy(ssid_null_terminated, params->ssid, WIFI_SSID_MAX_LEN);
+	ssid_null_terminated[params->ssid_length] = '\0';
```

Comment:
Is a check necessary to make sure that `params->ssid_length <= WIFI_SSID_MAX_LEN` ? (Similarly for psk_len)

Reply:
Yes need to add check length check for all layers. We also have separate ticket for that.

Reply:
I believe you are referring to the ticket which checks for `minimum` length. I was referring to the upper limit check here.